docs/sources/logql/template_functions.md

```diff
@@ -612,3 +612,40 @@ Example of a query to print a newline per queries stored as a json array in the
 ```logql
 {job="cortex/querier"} |= "finish in prometheus" | logfmt | line_format "{{ range $q := fromJson .queries }} {{ $q.query }} {{ end }}"
 ```
+
+## now
+
+`now` returns the current local time.
+
+```template
+{{ now }}
+```
+
+## toDate
+
+`toDate` parses a formatted string and returns the time value it represents.
+
+```template
+{{ toDate "2006-01-02" "2021-11-02" }}
+```
+
+## date
+
+`date` returns a textual representation of the time value formatted according to layout.
```

Comment:
Q: about the `layout`. Can we explain bit about it? What are the other options? (is it same as golang's datetime layout?, if yes, we can mention that and refer the doc to the layout?)

Reply:
This is a good question. I'm not actually sure, although I expected it was the same as golang's.

Reply:
Yes, you are correct, it is golang datetime layout, added doc references to the layout - https://github.com/grafana/loki/pull/4603/commits/2bd1a7194e9df1908a89054ee2c420e9b4812a3c

Reply:
Cool. thanks for adding the the clarification! :+1: